web/package.json

```diff
@@ -39,6 +39,7 @@
     "scripts": {
         "build": "npm run build --workspaces",
         "build:debug": "cross-env NODE_ENV=development CARGO_FEATURES=avm_debug npm run build",
+        "build:dual-wasm": "cross-env ENABLE_WASM_EXTENSIONS=true npm run build --workspaces",
```

Comment:
Also very petty, but `--workspaces` is not really needed here (see https://github.com/ruffle-rs/ruffle/pull/6025#discussion_r782749655).

Reply:
Okay, I have removed it.